### PR TITLE
Uploading mailroom assignments 1 and 2

### PR DESCRIPTION
**Pull request by tlhuman**

### CODE REVIEW

File: students/tim_lurvey/lesson03/mailroom_pt1.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+__author__ = 'Tim Lurvey, ig408c'
+
+import sys
+
+sys.path.append('C:\\Users\\ig408c\\Documents\\!SCHOOL\\!Python\\SP_Online_PY210-master\\students\\tim_lurvey')
+from myClasses import bunch
```
Comment (charlesrobison) [inline students/tim_lurvey/lesson03/mailroom_pt1.py]: Typically want your import statements together at the top of the file

File: students/tim_lurvey/lesson03/mailroom_pt1.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+__author__ = 'Tim Lurvey, ig408c'
+
+import sys
+
+sys.path.append('C:\\Users\\ig408c\\Documents\\!SCHOOL\\!Python\\SP_Online_PY210-master\\students\\tim_lurvey')
+from myClasses import bunch
+
+myData = {}
```
Comment (charlesrobison) [inline students/tim_lurvey/lesson03/mailroom_pt1.py]: Variable names should be snake_case in Python: my_data

File: students/tim_lurvey/lesson04/mailroom_pt2.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+__author__ = 'Tim Lurvey, ig408c'
+
+import sys
+
+sys.path.append('C:\\Users\\ig408c\\Documents\\!SCHOOL\\!Python\\SP_Online_PY210-master\\students\\tim_lurvey')
+from myClasses import bunch
```
Comment (charlesrobison) [inline students/tim_lurvey/lesson04/mailroom_pt2.py]: Same feedback for prior mailroom - for both import and variable names.

File: students/tim_lurvey/lesson03/mailroom_pt1.py (hunk 1 above)
Comment (tlhuman) [reply students/tim_lurvey/lesson03/mailroom_pt1.py]: Sorry, I had hard coded the sys path to myClasses.  I have the file in my /tim_lurvey path and my next pull will include the __init__.py to allow the code to be executed.  The class it really just a fancy dictionary, but it makes code easier to read.

**Pull request merged**